auth0/src/main/java/com/auth0/android/authentication/AuthenticationAPIClient.kt

```diff
@@ -379,17 +346,13 @@ public class AuthenticationAPIClient @VisibleForTesting(otherwise = VisibleForTe
      * Creates a user in a DB connection using ['/dbconnections/signup' endpoint](https://auth0.com/docs/api/authentication#signup)
      * and then logs in the user.
      * Example usage:
-     * <pre>
-     * `client.signUp("{email}", "{password}", "{username}", "{database connection name}")
-     * .start(new Callback<Credentials, AuthenticationException>() {
-     * {}Override
-     * public void onSuccess(Credentials payload) {}
-     *
-     * {}Override
-     * public void onFailure(AuthenticationException error) {}
-     * });DefaultClient(auth0.connectTimeoutInSeconds)
-    ` *
-    </pre> *
+     * ```
+     * client.signUp("{email}", "{password}", "{username}", "{database connection name}")
+     *     .start(object: Callback<Credentials, AuthenticationException>() {
```

Comment:
Here you are using Callback again. Let's be consistent on either Callback<T, U> or AuthenticationException<T>

Reply:
New commit to use `Callback` in docs.